tools/sort-test-expectations.mjs

```diff
@@ -92,14 +92,17 @@ for (let i = testExpectations.length - 1; i >= 0; i--) {
       testIdMatchesExpectationPattern(
         expectation.testIdPattern,
         candidate.testIdPattern
-      ) &&
-      isSubset(candidateParams, params) &&
-      isSubset(candidatePlatforms, platforms)
+      )
     ) {
       foundMatch = true;
```

Comment:
We need to mark that there is a match even if the other parts don't match.
```json
  {
    "testIdPattern": "[network.spec] network Network Events Page.Events.RequestServedFromCache",
    "platforms": ["darwin", "linux", "win32"],
    "parameters": ["chrome", "webDriverBiDi"],
    "expectations": ["PASS"]
  },
  ```
  Was being remove due to 
  ```json
    {
    "testIdPattern": "[network.spec] network Network Events *",
    "platforms": ["darwin", "linux", "win32"],
    "parameters": ["webDriverBiDi"],
    "expectations": ["FAIL"],
    "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
  },
  ```

Reply:
Aren't params needed for the match too?

Reply:
Yeah, I think the logic is wrong.
I think the `isSubset(candidateParams, params)` need to be reverse as we have an `AND` logic for Params

Reply:
I am not sure that is correct, reversing might remove a more specific expectations I think